Microsoft.Azure.Cosmos/src/Resource/QueryResponses/StandByFeedContinuationToken.cs

```diff
@@ -185,20 +189,19 @@ private void InitializeCompositeTokens(IEnumerable<CompositeContinuationToken> t
             this.currentToken = this.compositeContinuationTokens.Peek();
         }
 
-        private async Task<IReadOnlyList<Documents.PartitionKeyRange>> TryGetOverlappingRangesAsync(
-            Documents.Routing.Range<string> targetRange,
-            bool forceRefresh = false)
+        private async Task<IReadOnlyList<Documents.PartitionKeyRange>> GetOverlappingRangesAsync(
+            Documents.Routing.Range<string> targetRange)
         {
             Debug.Assert(targetRange != null);
 
-            IReadOnlyList<Documents.PartitionKeyRange> keyRanges = await this.pkRangeCacheDelegate(
+            IReadOnlyList<Documents.PartitionKeyRange> keyRanges = (await this.pkRangeCacheDelegate(
                 this.containerRid,
                 new Documents.Routing.Range<string>(
                     targetRange.Min,
                     targetRange.Max,
                     isMaxInclusive: false,
                     isMinInclusive: true),
-                forceRefresh);
+                forceRefresh: false)).Result;
```

Comment:
forceRefresh needs to be an input parameter, since the cache is optionally refreshed.
```suggestion
                forceRefresh: forceRefresh)).Result;
```